csharp-basics/exercises/Polymorphism/ScooterRental/Calculations.cs

```diff
@@ -0,0 +1,75 @@
+using ScooterRental.Exceptions;
+using ScooterRental.Interfaces;
+
+namespace ScooterRental;
+
+public class Calculations : ICalculations
+{
+    public decimal CalculateRentalPrice(DateTime start, DateTime end, decimal pricePerMinute)
+    {
+        const decimal maxPricePerDay = 20.0m;
+
+        if (start.Day == end.Day)
+        {
+            if ((decimal)(end - start).TotalMinutes * pricePerMinute > maxPricePerDay)
```

Comment:
Šeit if un zemāk aprēķinā rēķini 2x vienu un to pašu darbību. :) ieliec mainīgajā. :) ifā tad pārbaudi mainīgo. :) un ja nu kas apakšā nebūs jārēķina vēlreiz. :)